### ISSUE TITLE
:tests syntax refuses to run multiple tests

### ISSUE THREAD

**Issue by facundominguez** [commercialhaskell/stack#2149]
#### Steps to reproduce:

Create a multi-package project, each package containing a test suite with name **tests**.

Then run:
```
$ stack build  :tests
Error parsing targets: Ambiugous component name tests, matches: [(package-a,CTest "tests"),(package-b,CTest "tests")]
```

#### Expected:

The tests suites of both packages would be built and executed.

#### Actual:

stack refuses to build any tests. Instead one has to list all the packages explicitly:
```
stack build package-a:tests package-b:tests
```

**Comment by mgsloan** [comment]
Good point!  It'd particularly make sense with something like https://github.com/commercialhaskell/stack/issues/2155
